helm_deploy/calculate-journey-variable-payments/templates/_envs.tpl

```diff
@@ -94,12 +94,6 @@ env:
         name: {{ template "app.name" . }}
         key: HMPPS_AUTH_CLIENT_SECRET
 
-  - name: APPINSIGHTS_INSTRUMENTATIONKEY
```

Comment:
so if you remove that from the _envs.tpl, how you get the value injected?

Reply:
I see in the previous PR set up in APPLICATIONINSIGHTS_CONNECTION_STRING . so I do LGTM